src/SFA.DAS.FAA.Web/SFA.DAS.FAA.Web.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.StackExchangeRedis" Version="8.0.10" />
     <PackageReference Include="MediatR" Version="12.4.1" />
     <PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="17.1.113" />
-    <PackageReference Include="SFA.DAS.GovUK.Auth" Version="17.1.119" />
+	<PackageReference Include="SFA.DAS.GovUK.Auth" Version="17.1.124-prerelease-2" />

```

Comment:
just a minor note/reminder to update this with a non prerelease version

Reply:
@dashton82 SFA.DAS.GovUK.SignIn package upgraded

[SFA.DAS.GovUk.SignIn Package upgraded](https://github.com/SkillsFundingAgency/das-findapprenticeship-web/pull/245/commits/d81634f7fdc09a6b9feec185dcd0c07968f0e7c8)